packages/Webkul/Category/src/Repositories/CategoryRepository.php

```diff
@@ -294,6 +299,25 @@ public function getPartial($columns = null)
         return $trimmed;
     }
 
+    /**
+     * Get searched categories.
+     *
+     * @return \Illuminate\Support\Collection
+     */
+    public function getSearchedCategories()
+    {
+        return $this->model->scopeQuery(function ($query) {
+            return $query
+                ->select('categories.*')
+                ->leftJoin('category_translations', function ($query) {
+                    $query->on('categories.id', '=', 'category_translations.category_id')
+                        ->where('category_translations.locale', app()->getLocale());
+                })
+                ->where('category_translations.name', 'like', '%'.urldecode(request()->input('query')).'%')
```

Comment:
Use params variable here and remove request dependency.

$params['query'] and $params['limit']

Reply:
Sir done